packages/sdk/src/clients/skinnyOptimisticOracle/client.ts

```diff
@@ -84,6 +84,7 @@ export function reduceEvents(state: EventState, event: Event): EventState {
       const id = requestId(typedEvent.args);
       if (!state.requests) state.requests = {};
       state.requests[id] = {
+        ...(state.requests[id] || {}),
```

Comment:
bug 2, we needed to copy over existing found request data, otherwise it things like proposalTx requestTx got thrown out

Reply:
Why is the || needed?

Wouldn't `...state.requests[id],` work since undefined just expands the same as {} does?